### PR TITLE
Push "join URLs" into config

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

File: cmd/influxd/config.go [hunk 1]
```diff
@@ -46,6 +49,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Initialization struct {
+		JoinUrls string `toml:"joinUrls"`
```
Comment (benbjohnson) [inline cmd/influxd/config.go]: The other `toml` tags use dashes. Maybe `join-urls` instead?

Comment (otoolep) [inline cmd/influxd/config.go]: Makes sense.

Comment (otoolep) [inline cmd/influxd/config.go]: Fixed.

**Comment by otoolep** [comment]
+1 from @benbjohnson -- and green build. Merging now.

**Pull request merged**